searchconsole/query.py

```diff
@@ -399,3 +399,53 @@ def to_dict(self):
     def to_dataframe(self):
         import pandas
         return pandas.DataFrame(self.rows)
+
+
+class IndexStatus:  
+    def __init__(self, api):
+        self.api = api
+        self.raw = {}
+
+    def __repr__(self):
+        return "<searchconsole.index_status(url={})>".format(self.api.url)
+    
+    def urls(self, *urls):
+        list_urls = list(urls)
+        
+        if 'sc-domain' in self.api.url:
```

Comment:
Could you help me understand the need for this check?

I'm inclined to keep the cleaning minimal here and instead encouraging users ensure they are supplying valid URLs before calling this method.

Reply:
In fact, I'd probably recommend only accepting  a single URL (as the API does) and letting users handle iterating through a batch of URLs. That would save us having to implement the rate limiting and retry logic here. Perhaps that can come later but for simplicity I think it makes sense to just accept a single URL.

Reply:
In Google Search Console, you can have: 

1. **Domain property**: github.com 
2. **URL prefix**: www.github.com 

In `self.api.url`, the former is identified through a sc-domain prefix. Moreover, it allows you to get data for any subdomain linked to a given domain. So www.github.com, hello.github.com etc... 

While I understand why you'd want to accept only one URL (as the APi does), keep in mind that you can request only 2,000 URLs with this API (per day), hence should the code raise an error after the 100th iteration because the user doesn't have the right to inspect a given URL, 100 "credits" will be lost. That's why I wanted to introduce the check. 

But completely up to you obviously :)